### DISCUSSION TITLE
「Organizationが紐付いている記事の一括追加・削除」という文言について

### DISCUSSION THREAD

**Discussion by yamatai12** [increments/qiita-discussions#725] [要望・アイデア・不具合]
<img width="751" alt="image" src="https://github.com/increments/qiita-discussions/assets/123088250/217d2c40-83dc-4f5a-ac66-b519a0c2bf68">

いつもお世話になっています！
「Organizationが紐付いている記事の一括追加・削除」ですと記事自体が消えてしまうのかなと個人的に思ってしまいました。。

ご確認のほどよろしくお願いします🙏

**Comment by tanaka-asami415** [comment]
@yamatai12 
ご意見ありがとうございます。

> 「Organizationが紐付いている記事の一括追加・削除」ですと記事自体が消えてしまうのかなと

確かに分かりにくいですね。修正いたします！
教えていただきありがとうございます🙇

**Reply by yamatai12** [reply]
@tanaka-asami415 
お世話になります！
ご対応感謝いたします！
よろしくお願いします🙏

**Comment by kimkim0814** [comment]
@yamatai12 
お待たせしました。
Organization設定ページの文言の方修正いたしました!
具体的には、「削除」の表記を「解除」に変更し、誤解を招きにくい表記にしました。

この度はご意見をいただき、ありがとうございました！

**Reply by yamatai12** [reply]
@kimkim0814 
ご対応ありがとうございました！！
